Makefile

```diff
@@ -85,3 +85,6 @@ fmt: fmt_go fmt_tf
 .PHONY: clean
 clean:
 	rm -rf .terraform.d
+
+generate:
+	go generate ./...
```

Comment:
It seems like it missing a new line here at the end of the file.

Reply:
Nice catch thank you